### DISCUSSION TITLE
MediaKeys influence mpv-player although input-media-keys=no is set

### DISCUSSION THREAD

**Discussion by JackDjTom** [mpv-player/mpv#15449] [Q&A] [answered]
Hi,

i have set `input-media-keys=no` in my mpv.conf. But the media keys are still applied.
How can I achieve so the media keys never work on mpv-player whether its focused or unfocused/minimized?

**Comment by Akemi** [comment]
it would be useful to know on which platform you are on. also since this sounds like an actual issue, it is probably better to open an issue.

**Comment by guidocella** [comment, marked as answer]
`--media-controls=no`

**Reply by JackDjTom** [reply]
Thx so much!
